Cubical/Categories/Displayed/Poset.agda

```diff
@@ -0,0 +1,43 @@
+{-# OPTIONS --safe #-}
+
+module Cubical.Categories.Displayed.Poset where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.HLevels
+
+open import Cubical.Categories.Category.Base
+
+open import Cubical.Categories.Displayed.Base
+
+private
+  variable
+    ℓC ℓC' ℓCᴰ ℓCᴰ' : Level
+
+record Posetᴰ (C : Category ℓC ℓC') ℓCᴰ ℓCᴰ' :
```

Comment:
This should be `Preorderᴰ` since it doesn't require univalence/anti-symmetry

Reply:
Also should probably document somewhere that these are equivalent to categories with a faithful functor into C.

Reply:
Added the functor and the faithful proof from Grothendieck.agda to Displayed/Poset.agda.